examples/Constraints_Distributional_Measures.ipynb

```diff
@@ -389,43 +332,85 @@
   },
   {
    "cell_type": "markdown",
-   "id": "6a00e95c",
+   "id": "536f8a8e",
+   "metadata": {},
+   "source": [
+    "We are going to log the 'data_train' dataframe, to obtain a DatasetProfile for the training data. This profile will later be used to check for drift with the production data. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "29e7f093",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "profile = session.log_dataframe(data_train, \"test.data\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f943b3c4",
    "metadata": {},
    "source": [
-    "We present some simple and efficient ways to check for data quality and drift."
+    "Now, let's also log the 'data_test' dataframe, to obtain the target DatasetProfile for the comparison and for executing the data checks."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 7,
-   "id": "d082687d",
+   "id": "9c906fb0",
    "metadata": {},
    "outputs": [],
    "source": [
-    "def show_feature_bar_chart_in_train_and_test_data(feature, training_data, testing_data):\n",
-    "    fig, (ax1, ax2) = plt.subplots(1, 2, figsize=(20, 5))\n",
-    "    fig.suptitle(f'Distribution of the feature \"{feature}\"')\n",
-    "    training_data[feature].value_counts().plot.bar(ax=ax1)\n",
-    "    ax1.set_title(\"Training data set\")\n",
-    "    testing_data[feature].value_counts().plot.bar(ax=ax2)\n",
-    "    ax2.set_title(\"Testing data set\")\n",
-    "    plt.show()"
+    "test_profile = session.log_dataframe(data_test, \"test.data\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6a00e95c",
+   "metadata": {},
+   "source": [
+    "In the following subsections of this notebook, we present some simple and efficient ways to check for data quality and drift."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fce5bcc3",
+   "metadata": {},
+   "source": [
+    "For visual inspection of the data we are going to use whylogs' NotebookProfileViewer. The assumption is that visually exploring the production data each time a new batch arrives is too expensive, but for the purpose of this notebook and for better understanding of the constraints, we are going to include visual representation."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "da4101ea",
+   "metadata": {},
+   "source": [
+    "### Define the NotebookProfileViewer and set the target and reference profiles"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 8,
-   "id": "a7c4cee0",
+   "id": "dc5e40f6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from whylogs.viz import NotebookProfileViewer"
```

Comment:
The tests are going to fail since the implementation for the NotebookProfileViewer is not merged in mainline yet.